.github/workflows/test.yml

```diff
@@ -9,7 +9,7 @@ jobs:
       - uses: actions/checkout@v1
       - uses: actions/setup-node@v2
         with:
-          node-version: '14'
+          node-version: '18'
```

Comment:
Just double checking...did you actually mean to commit this change here?

Reply:
Yes, the workflow was failing without it https://github.com/api3dao/airnode-protocol-v1/actions/runs/4325151090/jobs/7550927220